unittests/test_logging.py

```diff
@@ -296,11 +296,12 @@ def test_multiple_handlers(self):
             'level': 'INFO',
             'handlers': [
                 {'type': 'stream', 'name': 'stderr'},
-                {'type': 'file', 'name': self.logfile}
+                {'type': 'file', 'name': self.logfile},
+                {'type': 'syslog'}
```

Comment:
Unfortunately, I don't think there is a very reliable way to unit test this. Just to ensure the code coverage, I would only add this handler if `/dev/log` exists.